### ISSUE TITLE
[Question] Do PneumaticCraft villagers open closed trades.

### ISSUE THREAD

**Issue by duncanwebb** [TeamPneumatic/pnc-repressurized#635]
### Minecraft Version
1.15.2

### Forge Version
31.2.36

### Mod Version
pneumaticcraft-repressurized-1.15.2-1.5.1-81

### Describe your problem, including steps to reproduce it
A couple of villagers have closed their trades, trading other items do not seem to make the villager reopen their trades or regain their health. The villagers are masters and I think it works up to this level.

Is this the correct behaviour?

**Comment by desht** [comment]
To be honest, I'm not sure.  The official wiki suggests that they can replenish their stocks (a couple of times per minecraft day) by working more at their job site, which is a Charging Station for pneumatic mechanics.  Do your villagers have access to Charging Stations?

**Comment by duncanwebb** [comment]
Yes placing a charging station near the PneumaticCraft villager opened his trades again.

What happened is that I placed a charging station on the outside wall of an unused house and a villager was converted to a PneumaticCraft villager and then for some reason he wandered off and a new villager became a PneumaticCraft villager. As soon as a second charging station was placed by the first villager the second villager appeared by the first villager. Now I placed down a third charging station by the second one. Let's see what happens.

They don't regain health with the trades, maybe they need a house too.
